src/main/resources/application.properties

```diff
@@ -1,28 +1,25 @@
 spring.application.name=fpl-ccd-data-migration-tool
 
-log.debug=false
-
 idam.api.url=http://localhost:4501
 idam.client.id=ccd_gateway
 idam.client.secret=OOOOOOOOOOOOOOOO
 idam.client.redirect_uri=http://localhost:3451/oauth2redirect
-idam.username=sam@hillingdon.gov.uk
+idam.username=hmcts-admin@example.com
```

Comment:
admin will only be allowed to migrated submitted cases - I guess this is awaiting system update user to replace?

Reply:
Yes, I am happy to revert that change as it will get replaced by system update user anyway.